config/samples/dataplane_v1beta1_openstackdataplane.yaml

```diff
@@ -135,5 +135,10 @@ spec:
           edpm_hosts_entries_vip_hosts_entries: []
           hosts_entries: []
           hosts_entry: []
+      networkAttachments:
+        - ctlplane
+        - internalapi
+        - storage
+        - tenant
```

Comment:
I noticed NETWORKATTACHMENTS were empty
```
oc get osaee -n openstack   
NAME                                            NETWORKATTACHMENTS   STATUS   MESSAGE
dataplane-deployment-configure-networkndjhw                          True     AnsibleExecutionJob complete
dataplane-deployment-configure-openstacktspcd                        True     AnsibleExecutionJob complete
dataplane-deployment-configure-osb58ch                               True     AnsibleExecutionJob complete
dataplane-deployment-install-openstackwl7qv                          True     AnsibleExecutionJob complete
dataplane-deployment-install-oswdn5w                                 True     AnsibleExecutionJob complete
dataplane-deployment-run-openstackpklrh                              True     AnsibleExecutionJob complete
dataplane-deployment-run-oswh795                                     True     AnsibleExecutionJob complete
dataplane-deployment-validate-network2z6vr                           True     AnsibleExecutionJob complete
```
@stuggi is this the right way to use `networkAttachments`?

Reply:
@fao89 right now the implementation in the dataplaneoperator for networkattachments [1] is only on a per node base. we probably want to enhance this and also add it as a per role, or on openstackdataplane top level parameter and pass it down.

[1] https://github.com/openstack-k8s-operators/dataplane-operator/pull/53